api/session/auth_session.py

```diff
@@ -90,7 +74,9 @@ def clear_session(return_app=None, return_path=None):
                 )
             else:
                 current_app.logger.warning("{return_app} not listed as a safe app.", extra=dict(return_app=return_app))
-                abort(400, "Unknown return app.")
+                resp = make_response({"detail": "Unknown return app."}, 400)
```

Comment:
Assume again that nothing programmatically consumes this, or connexion formats this into a `detail` attribute?

Reply:
connexion formatted it into a "detail" json thing yeah - this was tested, so the change makes the test pass